fe/fe-core/src/main/java/org/apache/doris/metric/DorisMetricRegistry.java

```diff
@@ -52,11 +50,11 @@ public synchronized void addSystemMetrics(Metric sysMetric) {
     }
 
     public synchronized List<Metric> getMetrics() {
-        return Lists.newArrayList(metrics);
+        return metrics.stream().sorted(Comparator.comparing(Metric::getName)).collect(Collectors.toList());
```

Comment:
why sort these metrics?  because it order may be not rigiht as below:

![image](https://user-images.githubusercontent.com/7404824/221464685-82081443-5d35-450b-be36-5f0e53d60664.png)